cmd/manager/server.go

```diff
@@ -30,11 +28,12 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
 
+	"istio.io/pkg/ctrlz"
+	"istio.io/pkg/log"
+
```

Comment:
import format

Reply:
What's the suggested format? Putting all istio related imports without blank line?

Reply:
yes I think istio related imports should be grouped together

Reply:
thanks!